medmodels/medrecord/indexers.py

```diff
@@ -653,7 +655,8 @@ def __setitem__(  # noqa: C901
                     )
 
             return None
-        return None
+        msg = "Should never be reached"
```

Comment:
nitpick: you added a space before the previous method (line 309), so this should also have an empty line

```suggestion
        
        msg = "Should never be reached"
```